tests/Functional/EndToEndTest.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace PhpTuf\ComposerStager\Tests\Functional;
+
+use Symfony\Component\Process\Process;
+
+/**
+ * @coversNothing
```

Comment:
Kinda makes me wish there was a `@coversEverything` tag, since that would probably be more accurate for an end-to-end test. =P